translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -1528,6 +1529,32 @@ Value *SPIRVToLLVM::transConvertInst(SPIRVValue *BV, Function *F,
   }
 }
 
+// Decide whether to set fast math flags on Val, which is usually a newly create
+// instruction. Decorations on BV may prevent us from setting some flags.
+void SPIRVToLLVM::maybeSetFastMathFlags(Value *Val, SPIRVValue *BV) {
```

Comment:
I prefer remove maybe in function name, looks good for others.

Reply:
I think setFastMathFlags() is enough.

Reply:
Done.

Reply:
Please merge the two commits. Theoretically, we make one commit in one pull request. So you can just update the original commit based on review comments.

Reply:
Done. (I thought you would "squash and merge" the pull request.)

Reply:
Oh, author takes this responsibility.